### PR TITLE
[OSCD] T1201: Password Policy Discovery - Linux

### PR DESCRIPTION
**Pull request by omergunal**
https://github.com/Neo23x0/sigma/issues/1011

Task 24

### CODE REVIEW

**Comment by zinint** [comment]
Hello and welcome!

Please help me to understand, are you planning to detect those via audit.d with the rule that's smth like this: (?)
``` bash
-w /etc/pam.d/ -p wa -k pam
```
Then you should probably use some additional `{field: value}`s in your detection, e.g.
``` YAML
service: auditd
detection:
    selection:
        type: 'PATH'
        name:
            - '/etc/pam.d/common-password'
```
for example, like here [rules\linux\auditd\lnx_auditd_alter_bash_profile.yml](https://github.com/Neo23x0/sigma/blob/master/rules/linux/auditd/lnx_auditd_alter_bash_profile.yml) or here [rules\linux\auditd\lnx_auditd_auditing_config_change.yml](https://github.com/Neo23x0/sigma/blob/master/rules/linux/auditd/lnx_auditd_auditing_config_change.yml).

What do you think? Did you run some tests? What did they show?

**Comment by zinint** [comment]
And also - `rules/windows/registry_event/sysmon_comhijack_sdclt.yml` - you already put this rule in PRs #1078 and #1079, and I already commented on it [there](https://github.com/Neo23x0/sigma/pull/1078#issuecomment-707334832). Please decide where you'll leave it and delete the other one.

**Comment by zinint** [comment]
Thanks for the changes, now linux rule is fine.

But you didn't do anything with the second rule in every one of your PRs `rules/windows/registry_event/sysmon_comhijack_sdclt.yml`:
1. Will you remake your PRs or leave it in everyone anyway?
2. Will ypu change the rule's `detection` according to the test wich [I described here](https://github.com/Neo23x0/sigma/pull/1078#issuecomment-707334832).

File: rules/linux/lnx_password_policy_discovery.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: Password Policy Discovery
+id: ca94a6db-8106-4737-9ed2-3e3bb826af0a
+status: stable
+description: Detects password policy discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/08
+references:
+    - https://attack.mitre.org/techniques/T1201/
```
Comment (yugoslavskiy) [inline rules/linux/lnx_password_policy_discovery.yml:8]: Hello @omergunal! Good to see your are back (:
Could you please remove the ATT&CK link and add a link to the relevant Atomic Red Team test here as well?

**Comment by yugoslavskiy** [comment]
Hello @omergunal! Thank you very much for your hard work! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**